api/queries_comments.go

```diff
@@ -139,6 +139,8 @@ func (c Comment) Reactions() ReactionGroups {
 	return c.ReactionGroups
 }
 
+// TODO: What is this supposed to do, and how does it relate to
+// formatting comments for PR or Issue view?
 func (c Comment) Status() string {
 	return ""
 }
```

Comment:
Looking through https://github.com/cli/cli/pull/2757, I believe `Comment` was originally an interface that encompassed issue comments, PR comments, and **PR reviews**.  It is that last one which would have various statuses that would be rendered via: 

https://github.com/cli/cli/blob/192f57ef429b4e89937c87dcbd330520730d4a3a/api/queries_pr_review.go#L115-L117

https://github.com/cli/cli/blob/192f57ef429b4e89937c87dcbd330520730d4a3a/pkg/cmd/pr/shared/comments.go#L166-L184